changelog/18463.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli/pki: Add List-Children functionality to pki client.
```

Comment:
Do we want to sync `List-Children` with `list-intermediates` in this changelog entry? I'm also wondering if it'd be better to remove all of these and add it as a "feature" or a single release note entry. But can ask others and deal with later.

Reply:
I've updated this changelog entry to make terminology interface-consistent for now.